data/docs/development-toc.yml

```diff
@@ -29,6 +29,8 @@ toc:
         url: /stamping
       - page: Health Rules
         url: /health-rules
+      - page: Events
```

Comment:
I'm not suuuper familiar with the flow, but, I imagine when cutting a non-development release we must make sure to add to the TOC of the new release, is that right?

I'm wondering if we should start a new PR for the a new release here in `cartographer-docs` so we make sure to not forget to add it / create a new issue for it

wdyt?

Reply:
Not 100% sure of the flow myself, although I did assume that when the next version is cut, that it would copy from development-toc.yaml